src/app/pages/details/instructor-resource-tab/partners/partners.js

```diff
@@ -34,7 +34,8 @@ export default function ({dataPromise, targetEl, bookAbbreviation}) {
                 description: partner.short_partner_description,
                 cost: partner.affordability_cost,
                 type: partner.partner_type,
-                url: `/partners?${partner.partner_name}`
+                url: `/partners?${partner.partner_name}`,
+                verifiedFeatures: partner.verified_features
```

Comment:
@RoyEJohnson - I can't tell if this is where you are pulling the field name but it should be `verified_by_instructor` from the backend